projects/aca-content/src/lib/components/details/details.component.spec.ts

```diff
@@ -128,4 +136,10 @@ describe('DetailsComponent', () => {
     fixture.detectChanges();
     expect(store.dispatch).toHaveBeenCalledWith(new SetSelectedNodesAction([node]));
   });
+
+  it('should call openAspectDialog and updateNodeAspects when the button is clicked', () => {
```

Comment:
That title looks invalid. You say that these 2 functions should be called when button is clicked... But I see that you just call openAspectDialog manually and test if updateNodeAspects was called. Looking at your title you should just trigger click event on that button and then test if updateNodeAspects. No need to test if openAspectDialog is called and mentioning it in title.

Reply:
Updated test case title